BrainPortal/app/controllers/tasks_controller.rb

```diff
@@ -1502,6 +1502,7 @@ def handle_preset_actions #:nodoc:
       if (! preset_id.blank?) && preset = CbrainTask.where(:id => preset_id, :status => [ 'Preset', 'SitePreset' ]).first
         old_params = @task.params.clone
         @task.params         = preset.params
+        @task.description   += @task.description.blank? ? "#{preset.description}" : "\n\nDescription from preset configuration:\n\n#{preset.description}"
```

Comment:
This will crash if @task.description is nil

Reply:
This is ok except in the case where the preset doesn't have a description. Then the result is that we get just 'Description from preset configuration'